.github/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+## Summary
+
+<!--
+A few sentences describing the changes being proposed in this pull request.
+-->
+
+## List of notable changes:
+
+<!--
+E.g.
+
+- **added** # for # component because #
+- **removed** props for # component because #
+- **updated** documentation for # component because #
```

Comment:
```suggestion
- **added** new design token for # because #
- **deprecated**  design token for # because #
- **updated** documentation for # because #
```